packages/core-ethereum/src/index.ts

```diff
@@ -147,7 +171,10 @@ export default class HoprEthereum extends EventEmitter {
    * Retrieves ETH balance, optionally uses the cache.
```

Comment:
Where is the bespoke option?

Reply:
Not sure I follow here. Also, nothing was changed re: the class or methods.